### DISCUSSION TITLE
Answer to review comments on own PR

### DISCUSSION THREAD

**Discussion by andy941** [pwntester/octo.nvim#610] [Q&A]
Hi all, loving the plugin so far.

One thing I would love to do since we can already have a diff with locally checked out files would be to use Octo to answer reviewers comments from neovim. This will include not only resolving/answering threads but crucially making codes changes and push them. Is that already possible? If so, what would be the "best" workflow?

Thank you,
Andrea

**Comment by GuillaumeLagrange** [comment]
It is already possible, the caveat is you need to start a review of your own in order to enter the diff view, and display comments and answer to them.

We could think about a dedicated view, which would open some sort of quickfix window at the bottom and allow to quickly jump between unresolved thread comments.

**Reply by andy941** [reply]
That would be amazing. Basically the same as the current review files window but when I press enter on an entry it just opens the local file, with comment thread on the side would already be amazing. 

One other annoying thing is to commit, copy and paste the commit ID and resolve all comments that way. Usually it's the same commit where I answer all comments from a reviewer or multiple ones so I am copying and pasting the same commit ID everywhere. On GitHub webui it's painful to say the least, would be nice to streamline it in Octo in some way.

**Reply by GuillaumeLagrange** [reply]
For the commit stuff, IMO when we open the comment

- If the comment is on a ref that is locally checked out and `use_local_fs` is used, we open the local file buffer
- else we open an octo buffer, exactly how we do it in the diff view for review

An issue would be better suited to fine tune the details. Could you open one with how you would like the feature to behave ? I cannot guarantee anything timeline wise, but once I'm done working on #601 I can have a look, or give you pointers if you wanna hack at it.
